frontend/tests/e2e/features/common/index.ts

```diff
@@ -103,13 +103,6 @@ Then('sees the register form', () => {
   cy.screenshot();
 });
 
-Then('sees success messages', () => {
-  cy.get('span.success-message-bottom').should('exist');
-
-  cy.wait(5500);
-  cy.get('span.success-message-bottom').should('not.exist');
-});
```

Comment:
If you feel like this is useful, I would try to provide an specific test for this behavior. (prefer jest test intead for this).